src/source-pcap-file-helper.c

```diff
@@ -83,9 +135,16 @@ void PcapFileCallbackLoop(char *user, struct pcap_pkthdr *h, u_char *pkt)
     p->pcap_cnt = ++pcap_g.cnt;
 
     p->pcap_v.tenant_id = ptv->shared->tenant_id;
+    p->pcap_v.pfv = ptv;
     ptv->shared->pkts++;
     ptv->shared->bytes += h->caplen;
 
+    p->ReleasePacket = PcapFileReleasePacket;
+
+    /* Inform capture-neutral hook about packet creation so capture
+     * implementations can attach context without generic code changes. */
+    CaptureHooksOnPseudoPacketCreated(p);
```

Comment:
why is this called here? This is not the place where a pseudo packet is created, right?